### PR TITLE
Improve homepage copywriting

### PR DESCRIPTION
**Pull request by MattiSG**
Also happens to fix by coincidence a minor layout issue.

## Before

Capture d’écran 2019-06-11 à 13 26 38 1

## After

Capture d’écran 2019-06-11 à 13 27 16

### CODE REVIEW

**Comment by Ndpnt** [comment]
GTM!

File: _pages/en/encyclopedia/index.md [hunk 1]
```diff
@@ -6,32 +6,32 @@ nav_order: 1
 numbered_headers: false
 ---
 
-## If you're reading this, you're probably trying to counter disinformation.
+## If you’re reading this, you’re probably trying to counter disinformation.
 {: .thin}
```
Comment (Ndpnt) [inline _pages/en/encyclopedia/index.md]: I think, this is the good PR to move `{: .thin}` to the second header : "Good news! You’re not alone."
```
## If you’re reading this, you’re probably trying to counter disinformation.
## Good news!<br />You’re not alone.
{: .thin .mt-0}
```

Comment (MattiSG) [reply _pages/en/encyclopedia/index.md]: Sounds good. Would be best if you had a screenshot, but otherwise I trust you.

Comment (Ndpnt) [reply _pages/en/encyclopedia/index.md]: Capture d’écran 2019-06-11 à 16 12 55

Comment (MattiSG) [reply _pages/en/encyclopedia/index.md]: Lovely, go for it!

**Pull request merged**